go/extractor/project/project.go

```diff
@@ -193,7 +193,7 @@ func findGoModFiles(root string) []string {
 }
 
 // A regular expression for the Go toolchain version syntax.
-var toolchainVersionRe *regexp.Regexp = regexp.MustCompile(`(?m)^([0-9]+\.[0-9]+\.[0-9]+)$`)
+var toolchainVersionRe *regexp.Regexp = regexp.MustCompile(`(?m)^([0-9]+\.[0-9]+(\.[0-9]+|rc[0-9]+))$`)
```

Comment:
I don't think this needs to wait until Go 1.24 is out. Have/are you going to move this to its own PR?

Reply:
I agree it could be in a separate PR. I wasn't going to do that, since there's some overhead in doing it. Do you think I should?

Reply:
I think it's worth doing, and it shouldn't be that much overhead to cherry-pick this commit into its own branch/PR?

Reply:
Go 1.24.0 has been released, so this PR is ready now. No point in making this commit a separate PR any more.